plugins/view-mapped/src/view-mapped.ts

```diff
@@ -56,6 +58,12 @@ export function viewMappedPlugin() {
 
             mappedFields.push(destinationFieldKey)
           }
+
```

Comment:
NIT :
await api.jobs.ack(jobId, ...)

is repeated multiple times, perhaps it can be combined as a helper method?

// Helper method for acknowledging job progress
async function updateJobProgress(jobId, info, progress) {
  await api.jobs.ack(jobId, {
    info,
    progress,
  });
}

And be called 

await updateJobProgress(jobId, "Updating the table to only view mapped fields...", 30);

await updateJobProgress(jobId, "Halfway there, hang tight...", 50);

await updateJobProgress(jobId, "Almost done...", 80);